db/people.yml

```diff
@@ -2040,3 +2040,263 @@ zzgong:
   service: []
   skills: []
   title: Dr.
+rdifrancesco:
+  active: false
+  aka:
+    - Remus DiFrancesco
```

Comment:
I never heard him having this aka.  Do you know something I don't?  Or maybe just remove.  If there is an empty list here it will be ``aka: []``